lib/generators/active_admin/assets/assets_generator.rb

```diff
@@ -6,7 +6,7 @@ class AssetsGenerator < Rails::Generators::Base
 
       def install_assets
         template 'active_admin.js.coffee', 'app/assets/javascripts/active_admin.js.coffee'
-        template 'active_admin.css.scss',  'app/assets/stylesheets/active_admin.css.scss'
+        template 'active_admin.scss',  'app/assets/stylesheets/active_admin.scss'
```

Comment:
Line is too long. [81/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.